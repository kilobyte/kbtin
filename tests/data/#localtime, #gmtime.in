#rem #localtime foo 1189969300;#rem #showme $foo
#rem output for CET was: 40 01 21  16 09 2007  0 259 1
#gmtime foo 1189969300;#showme $foo
