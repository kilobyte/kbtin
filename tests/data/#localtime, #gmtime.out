40 01 19  16 09 2007  0 259 0
